.github/workflows/flutter_build.yml

```
@@ -0,0 +1,38 @@
+name: Build Flutter App
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)